app/screens/home/search/results/file_card.tsx

```diff
@@ -0,0 +1,139 @@
+            flexDirection: 'row',
+
+            marginHorizontal: 20,
+            marginVertical: 4,
+            paddingVertical: 8,
+
+            borderWidth: 1,
+            borderRadius: 4,
+            borderColor: changeOpacity(theme.centerChannelColor, 0.16),
+
+            shadowColor: changeOpacity(theme.centerChannelColor, 0.16),
+            shadowOffset: {
+                width: 0,
+                height: 2,
+            },
+            shadowOpacity: 0.8,
+            elevation: 1,
+        },
+        flex: {
+            column: {flexDirection: 'column'},
+            row: {flexDirection: 'row'},
+        },
+        iconContainer: {
+            marginHorizontal: 8,
+        },
+        nameText: {
+            color: theme.centerChannelColor,
+            ...typography('Heading', 200, 'SemiBold'),
+        },
+        infoText: {
+            color: changeOpacity(theme.centerChannelColor, 0.64),
+            ...typography('Body', 75, 'Regular'),
+        },
+        channelText: {
+            marginRight: 4,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+            borderRadius: 4,
+            paddingHorizontal: 4,
+        },
+        dateText: {
+            marginBottom: 2,
+        },
+        threeDotContainer: {
+            flex: 1,
+            alignItems: 'flex-end',
+        },
+        threeDot: {
+            top: 13,
+            right: 20,
+        },
+    };
+});
+
+export default function FileCard({fileInfo}: Props) {
+    const theme = useTheme();
+    const style = getStyleSheet(theme);
+    const size = getFormattedFileSize(fileInfo.size);
+    const format = 'MMM DD HH:MM A';
+
+    // TODO: hook this up
+    const openGallery = () => {
+        /* eslint-disable no-console */
+        console.log('Open Gallery');
+    };
+
+    // TODO: hook this up
+    const handleThreeDotPress = () => {
+        /* eslint-disable no-console */
+        console.log('pressed 3 dot');
+    };
+
+    // TODO: Get the channel name and display
+    const channelName = () => { // eslint-disable-line @typescript-eslint/no-unused-vars
+        return 'channelName';
+    };
+
+    // TODO: Get the filetype and display
+    const fileType = () => { // eslint-disable-line @typescript-eslint/no-unused-vars
+        return 'fileType';
+    };
+
+    return (
+        <TouchableOpacity
+            onPress={openGallery}
+            style={style.container}
+        >
+            <View style={style.iconContainer}>
+                <FileIcon
+                    file={fileInfo}
+                    iconSize={48}
+                />
+            </View>
+            <View style={style.flex.column}>
+                <Text style={style.nameText}>{fileInfo.name}</Text>
+                <View style={style.flex.row}>
+                    {/* <Text style={[style.infoText, style.channelText]}>{channelName()}</Text> */}
+                    {/* <Text style={style.infoText}>{fileType()}</Text> */}
```

Comment:
Why keep this commented if you have already the placeholders?

Reply:
If it is going to ship in the beta, I agree we may want to skip the placeholders (and I would remove them completely, but maybe it is just me).

But if the plan is to rethink the design, I think is valuable to add here relevant placeholders for @matthewbirtch to see. I am curious about two things: What is the longest name we plan to put for the file type? Are we talking about being the longest something like `JPEG` or `TEXT`, or are we planning on something like `Spreadsheet` or `Image`? How will this look with a "long channel name" (sixty-four W (i.e. `WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW`))? How long is going to be the size number? (I guess it should go into MB when it is over 1000, and so on, so the longest will be `999 XB`, but 1000 KB is not 1MB, so not sure if it can show `1000 KB`) And what about a extra long file name?

Reply:
☝️

Reply:
This is another reason I left the placeholders until we have a resolution.  As you can see here, the combinabtion of a long channel name, just by itself creates issues. 
<img width="543" alt="image" src="https://user-images.githubusercontent.com/7575921/174416241-71fb155c-8ab6-4d32-a71a-ce8df0c1d2d6.png">

Reply:
Perhaps we could add a third line which will grow the list, or truncate the channel name if it is over a predetermined number of characters.

Reply:
right, but then we should add some ellipsis the channel name with a maxWidth.. the ellipsis should also apply to the file name, thus the extension of the file may not show..  but yeah will defer to UX

Reply:
This would be outside of the scope of this PR, but I have some other thoughts about the search in general.

1) if the channel name was also touchable, the user could click the channel name in this card to add a channel filter.

2) I think a really cool feature would be an advanced search feature that opens a filter screen (similar to the file options screen). Inside it would be the filter terms as text inputs with autocomplete and date pickers.  Instead of populating the search box in the navigation, these terms woudl be shown as terms under the `Messages` and `Files` header and include an `x` to remove the term. 

I think this would be nice because the search in the navigation would only contain the text a person is searching for and not all the terms. Those would be included in the call to the server, but would not need to be in the search text box.  

I don't think this would be too difficult to create and would make search much more pleasant on mobile.  I'd love to discuss or put together a demo sometime.

Reply:
Here's a quick mockup of what I was trying to describe.
<img width="466" alt="image" src="https://user-images.githubusercontent.com/7575921/174417041-8e993992-362d-4ea9-8090-ccf6149fbc40.png">

Reply:
I also think any file with a date not in the current year should have the `Year` added in the card.

Reply:
> But if the plan is to rethink the design, I think is valuable to add here relevant placeholders for @matthewbirtch to see. I am curious about a few things: What is the longest name we plan to put for the file type? Are we talking about being the longest something like `JPEG` or `TEXT`, or are we planning on something like `Spreadsheet` or `Image`?

This is intended to be the file extension, so 4 characters

> Perhaps we could add a third line which will grow the list, or truncate the channel name if it is over a predetermined number of characters.

No, sorry, we don't want to add a 3rd row. We'd be better off cutting something out (like the date/time) rather than adding a 3rd row

Reply:
> And what about a extra long file name?

Should be truncated to 1 line